release.nix

```diff
@@ -80,6 +80,8 @@ in commonLib.nix-tools.release-nix {
     "nix-tools.tests.ouroboros-network.tests.x86_64-darwin"
     # 'Storage.HasFS.HasFS' test failing:
     "nix-tools.tests.ouroboros-consensus.test-storage.x86_64-darwin"
+    # doctests tests don't seem to work well on hydra
```

Comment:
You know that this is just an argument for turning them into regular unit/property tests :grin:

Reply:
I genuinely don't like having such things for very internal functions like this one. They force exporting the function, which makes it part of the public interface. Also the context is missing. Having the examples in the documentation actually tells you something about how the code behaves.

Reply:
I understand your argument, but not running the tests on CI feels wrong. Ideally, we get the doctests running on Hydra. Maybe you can copy the approach taken here https://github.com/input-output-hk/cardano-ledger-specs/search?q=doctest&unscoped_q=doctest

Reply:
Ironically I avoided doctest-discover because I've had problems getting it to play nicely with hydra in the past, but will prod it a bit...